recipes/databricks-cli/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "databricks-cli" %}
+{% set version = "0.8.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9db04827ded11313892b7f190f95af00677b91743c4b7009e1094f7f1ffc6bc7
+
+build:
+  number: 0
+  entry_points:
+    - databricks=databricks_cli.cli:cli
+    - dbfs=databricks_cli.dbfs.cli:dbfs_group
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - pip
+    - python
+    - setuptools
+  run:
+    - click >=6.7
+    - configparser >=0.3.5
+    - python
+    - requests >=2.17.3
+    - six >=1.10.0
+    - tabulate >=0.7.7
+
+test:
+  imports:
+    - databricks_cli
+    - databricks_cli.clusters
+    - databricks_cli.configure
+    - databricks_cli.dbfs
+    - databricks_cli.groups
+    - databricks_cli.jobs
+    - databricks_cli.libraries
+    - databricks_cli.runs
+    - databricks_cli.sdk
+    - databricks_cli.secrets
+    - databricks_cli.stack
+    - databricks_cli.workspace
+    - integration
+    - integration.dbfs
+    - integration.workspace
+  commands:
+    - databricks --help
+    - dbfs --help
+
+about:
+  home: https://github.com/databricks/databricks-cli
+  license: Apache 2.0
+  license_family: APACHE
+  license_file: LICENSE
```

Comment:
Please change this to license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE' when manually packaging the license ... see also https://conda-forge.readthedocs.io/en/latest/meta.html#packaging-the-license-manually